src/ui/units/wizard/containers/Wizard/WizardActionPanel/WizardActionPanel.tsx

```diff
@@ -88,12 +90,22 @@ export const WizardActionPanel: React.FC<WizardActionPanelProps> = (
         }
     }, [isCurrentRevisionActual, onSaveCallback]);
 
+    const canGoBack = useSelector<DatalensGlobalState, ReturnType<typeof selectCanGoBack>>(
```

Comment:
Should we add selectors to separate file?

Reply:
We always use `useSelector` in component file...